### ISSUE TITLE
Include Network ID and Record ID on Activity Stream

### ISSUE THREAD

**Issue by robgpeak** [ChrisASearles/CompLinks#121]
@cgladue I would be helpful to include the Advertiser's Network ID in the table in its own column, especially since we change names of advertisers.  Additionally, I would leave the Record ID column visible all the time. It is useful to see, and I would have it visible in all the filters as well. Meaning, even if I am in "New Ads" tab, it would be great to have both of those columns. 

When I refresh the page now, the Record ID's are visible for a moment, but then they disappear.

**Comment by cgladue** [comment]
ok, i assumed the Activity ID wouldnt be useful to you, and is only used to determine which activities to bulk update, but if you want to see it at all times then i can.  and when you say network ID you mean the ID that CJ or LS referrs to them as right ?  not our internal Advertiser ID

**Comment by robgpeak** [comment]
Yes on the Activity ID being visible all the time, and yes on using the LS and CJ network IDs, not our internal Advertiser ID
